app/controllers/external_users/claims_controller.rb

```diff
@@ -16,6 +18,7 @@ class ExternalUsers::ClaimsController < ExternalUsers::ApplicationController
 
   before_action :set_and_authorize_claim, only: %i[show edit update unarchive clone_rejected destroy summary
                                                    confirmation show_message_controls messages disc_evidence]
+  before_action :redirect_unless_editable, only: %i[edit update]
```

Comment:
Previously this was only checked for the `:edit` action, anyone that tried to update the claim directly using any HTTP client would be able to update it. This prevents that and keeps the behaviour in line between `edit` and `update`